tests/std/tests/P0355R7_calendars_and_time_zones_io/test.cpp

```diff
@@ -221,6 +221,29 @@ void test_limits(const char* flag, const IntType min, const IntType max) {
     assert(value == TimeType{max});
 }
 
+void test_lwg_3536() {
+    // LWG-3536, "Should chrono::from_stream() assign zero to duration for failure?"
+    minutes mm{20};
+
+    {
+        istringstream iss{"2:2:20"};

```

Comment:
This test is correct, but I have a clarity nitpick. When choosing values to use in tests, I believe that choosing distinct values for distinct purposes makes it easier to understand what the test is doing. Here, `20` is the initial value of `minutes mm`, which the test expects to remain unchanged after parse failure. This input reuses the value `20` for the number of seconds being parsed, but there is no relationship between the two - it's the mere presence of non-zero seconds that causes the parse to fail. Changing this to any other non-zero value, like `30`, would (very slightly) improve clarity.